UPGRADE-14.0.md

```diff
@@ -764,10 +764,83 @@ Follow the instructions in relevant sections, e.g. `shopsys/coding-standards` or
     -   Frontend API `privacyPolicyArticle` query now returns `ArticleSite` type instead of `Article`
     -   Frontend API `cookiesArticle` query now returns `ArticleSite` type instead of `Article`
     -   see #project-base-diff to update your project
--   fix broken drag and drop in GrapesJS in Safari ([#2966])(https://github.com/shopsys/shopsys/pull/2966)
+-   fix broken drag and drop in GrapesJS in Safari ([#2966](https://github.com/shopsys/shopsys/pull/2966))
     -   see #project-base-diff to update your project
 -   add Category ID to CategoryHierarchyItem ([#2962](https://github.com/shopsys/shopsys/pull/2962))
     -   see #project-base-diff to update your project
+-   add hreflang feature ([#2970](https://github.com/shopsys/shopsys/pull/2970))
+    -   method `Shopsys\FrameworkBundle\Model\Blog\Article\Elasticsearch\BlogArticleExportQueueFacade::__construct()` changed its interface:
+    ```diff
+        public function __construct(
+            Redis $redisQueue,
+    +       protected readonly BlogArticleFacade $blogArticleFacade,
+    ```
+    -   method `Shopsys\FrameworkBundle\Model\Blog\Article\Elasticsearch\BlogArticleExportRepository::__construct()` changed its interface:
```

Comment:
`BlogArticleExportRepository` was added in this version, so no upgrade notes are needed here, am I right?

Reply:
Right! removed